static/app/views/releases/detail/overview/releaseComparisonChart/releaseEventsChart.tsx

```diff
@@ -117,45 +119,79 @@ function ReleaseEventsChart({
   const environments = location.query.environment;
 
   return (
-    <EventsChart
-      query={getQuery()}
+    /**
+     * EventsRequest is used to fetch the second series of Failure Rate chart.
+     * First one is "This Release" - fetched as usual inside EventsChart
+     * component and this one is "All Releases" that's shoehorned in place
+     * of Previous Period via previousSeriesTransformer
+     */
+    <EventsRequest
+      organization={organization}
+      api={new Client()}
+      period={period}
+      project={projects}
+      environment={environments}
```

Comment:
environments will always only contain one enviroment?

Reply:
It can be multiple. I've noticed that `environment` prop accepts array too (even though it's singular).